js/interface-lists.js

```diff
@@ -510,9 +510,10 @@ function attahObservers() {
   });
 
   Fliplet.Widget.onSaveRequest(function () {
-    if (dynamicLists.isLoaded) {
-      $('form').submit();
+    if (!dynamicLists.isLoaded) {
+      Fliplet.Widget.complete();
```

Comment:
@MaksymShokin Can you explain why you haven't stopped the code execution with `return` after this? Otherwise, it looks like if the configuration isn't loaded, it will still run `$('form').submit()`, which would replicate the reported bug.

Reply:
@tonytlwu When I was testing this code on staging it worked fine the way it is. If you insist I can add an extra return to be safe.

Reply:
@MaksymShokin I imagine it's probably because `Fliplet.Widget.complete()` is synchronous and closes the pane immediately. Let's add a `return` to be clearer for future developers that the execution should be stopped there.

Reply:
@tonytlwu Fixed